frontend/src/API/use/useInterpreter.ts

```diff
@@ -120,7 +128,7 @@ export function useInterpreter(originalContent: string, originalTitle: string |
             const rect = contentRef.current.getBoundingClientRect();
             const topbarHeight = document.getElementById('topbar')?.clientHeight;
 
-            if (currentMode === 'annotate') {
+            if (currentMode === 'altTranslate' || currentMode === 'annotate') {
```

Comment:
Actually we should avoid scrolling to the beginning of the comment if we are adding content right above the button use has just clicked

Reply:
When the content is added to the bottom, it's scrolling to the bottom. This is needed in case when autocat is expanded (although I'm not sure if it works well).